### ISSUE TITLE
Resource Voting or Rating 🤔

### ISSUE THREAD

**Issue by Perry-Bright** [GDG-On-Campus-ICTU/MaterialHub-Hactoberfest#18]
We can implement a voting or rating system for materials to help surface the most valuable resources

**Comment by mohtasheem135** [comment]
Yes, this one is interesting we can also add a filter like "most favourite".

**Comment by Perry-Bright** [comment]
Yeah, or more like "Most Used" or something like that

**Comment by kmahesh18** [comment]
assign me this... i'll add likes to materials and filter accordingly
